include/AutomationEditor.h

```diff
@@ -240,6 +240,8 @@ protected slots:
 
 	EditModes m_editMode;
 
+	bool m_mouseDownLeft; //true if left click is being held down
```

Comment:
Agreed, will remove this one. Also, suggested change(s) might be applicable to PianoRoll as it is never used also.

https://github.com/LMMS/lmms/blob/85ed63be852d4da2555da49f93d8fd0e18cc3f6c/src/gui/editors/PianoRoll.cpp#L1925

(Only line where it is used)